cli/tests/unit/data/node.txt

```diff
@@ -1,2 +1,2 @@
-HOSTNAME       IP                          ID                    
-dcos-01   172.17.8.101  20150630-004309-1695027628-5050-1649-S0  
\ No newline at end of file
+HOSTNAME       IP                          ID                    TYPE   
+dcos-01   172.17.8.101  20150630-004309-1695027628-5050-1649-S0  slave  
```

Comment:
Ah, i overlooked this earlier, for TYPE we should say leader/agent not master/slave. We've been replacing the terminology. Quick change and then lets ship this!